server/jetstream_cluster_3_test.go

```diff
@@ -356,7 +356,7 @@ func TestJetStreamClusterDeleteConsumerWhileServerDown(t *testing.T) {
 
 	// Restart.
 	s = c.restartServer(s)
-	checkFor(t, time.Second, 200*time.Millisecond, func() error {
+	checkFor(t, time.Second*2, 200*time.Millisecond, func() error {
```

Comment:
just a nit, don't have to pick particularly for this PR 
```suggestion
	checkFor(t, 2*time.Second, 200*time.Millisecond, func() error {
```